dotnet/samples/NoCodeGenServer/Program.cs

```diff
@@ -0,0 +1,136 @@
+﻿// See https://aka.ms/new-console-template for more information
+using System.Diagnostics;
+using Azure.Iot.Operations.Mqtt.Session;
+using Azure.Iot.Operations.Protocol.Connection;
+using Azure.Iot.Operations.Protocol.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+// USE SDK classes without code-gen to provide feedback to the team
+
+// setup logging
+using var loggerFactory = LoggerFactory.Create(builder => builder.AddConsole());
+var logger = loggerFactory.CreateLogger("NoCodeGenServer");
+
+// load configuration
+IConfiguration configuration = new ConfigurationBuilder()
+    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+    .AddEnvironmentVariables()
+    .AddCommandLine(args)
+    .Build();
+
+// cancellation support
+var cancellationTokenSource = new CancellationTokenSource();
+Console.CancelKeyPress += delegate
+{
+    logger.LogInformation("SIGINT detected - closing application");
+    cancellationTokenSource.Cancel();
+};
+
+var sessionClientOptions = new MqttSessionClientOptions
+{
+    EnableMqttLogging = true,
+};
+
+var mqttClient = new MqttSessionClient(sessionClientOptions);
+
+var connectionString = configuration.GetConnectionString("Default");
```

Comment:
We have moved away from connection string in our samples.

Let align with using the .env file: https://github.com/Azure/iot-operations-sdks/blob/main/dotnet/samples/README.md#run-a-sample

and then load from environment variables:
```
        MqttConnectionSettings settings = MqttConnectionSettings.FromEnvVars();
```